src/test/java/com/hocs/test/pages/draft/Draft.java

```diff
@@ -291,6 +291,7 @@ public void assertHowDoYouIntendToRespondErrorMessage() {
     }
 
     public void assertPleaseSummariseYourCallErrorMessage() {
+        sleep(500);
```

Comment:
Is there a difference between sleep and wait a bit? If not should we stick to one of them?

Reply:
They're the same. I must have put `sleep` by mistake here.